### ISSUE TITLE
May I generate the evaluate csv?

### ISSUE THREAD

**Issue by pokameng** [MCG-NJU/MultiSports#5]
你好！我现在已经得到了我的训练模型。按照比赛要求，是不是要自己生成test.pkl里面每一帧的boundingbox？也就是x1,y1,x2,y2?

**Comment by yixuanli98** [comment]
Yes, more submission format details are in the 'Learn the Details/Submissions' part of the competition [https://competitions.codalab.org/competitions/33355#learn_the_details-submissions](https://competitions.codalab.org/competitions/33355#learn_the_details-submissions)
